lib/actions/ExtensionAction.js

```diff
@@ -74,7 +75,7 @@ class ExtensionAction {
     const files = fsEx.readdirSync(EXTENSION_FOLDER)
     const exts = []
     files.forEach(file => {
-      if (this.getExtensionId(file)) exts.push(file)
+      if (!file.startsWith('.') && this.getExtensionId(file)) exts.push(file)
```

Comment:
this check for "startsWith" should be obsolete, I suggest we just check each "file" whether or not its a "folder" and if it is try to load its extension config
alternative: use e.g. glob to only get "dirs" in the first place instead of files